File: engine/engine-model/src/main/scala/tools/packet/character/box/RemoveMiniGameBox.scala
package tools.packet.character.box

import net.opcodes.SendOpcode
import tools.packet.PacketInput

class RemoveMiniGameBox(private var _characterId: Int) extends PacketInput {
  def characterId: Int = _characterId

  override def opcode(): SendOpcode = SendOpcode.UPDATE_CHAR_BOX
}

File: engine/engine-model/src/main/java/tools/packet/character/box/RemoveMiniGameBox.java
package tools.packet.character.box;

import net.opcodes.SendOpcode;
import tools.packet.PacketInput;

public record RemoveMiniGameBox(Integer characterId) implements PacketInput {
   @Override
   public SendOpcode opcode() {
      return SendOpcode.UPDATE_CHAR_BOX;
   }
}